Content.Shared/Gibbing/Components/GibbableComponent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Content.Shared.Gibbing.Systems;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Gibbing.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, Access(typeof(GibbingSystem))]
+public sealed partial class GibbableComponent : Component
+{
+    [DataField(required:true), AutoNetworkedField, ViewVariables(VVAccess.ReadWrite)]
+    public List<EntProtoId> GibPrototypes = new();
+
+    [DataField(required:true), AutoNetworkedField, ViewVariables(VVAccess.ReadWrite)]
+    public int GibCount = 3;
+
+    [DataField, AutoNetworkedField, ViewVariables(VVAccess.ReadWrite)]
+    public SoundSpecifier? GibSound = new SoundCollectionSpecifier("gib");
+
+    public const float GibScatterRange = 0.3f;
```

Comment:
Gib scattering is the previous implementation and it had a fixed distance it would scatter drops. This isn't used for the physics based gibbing, its for when you just drop the items on the ground. 
This value only exists to make sure that all the items don't get placed in a single spot overlapping each other.